sdk/src/main/java/com/meniga/sdk/MenigaSettings.java

```diff
@@ -133,17 +134,17 @@ public CustomErrorHandler getErrorHandler() {
 	 *
 	 * @return The map containing the mapping between model types and their endpoints
 	 */
-	public Map<Service, String> getSpecialServiceEndpoints() {
+	public Map<Service, SpecialServiceEndpointDefinition> getSpecialServiceEndpoints() {
 		return specialServiceEndpoints;
 	}
 
 	/**
-	 * Gets the OkHttp read and write timeout limit in ms
+	 * Gets the OkHttp read and write timeoutInSeconds limit in ms
```

Comment:
Was the javadoc wrong in the first place? It was seconds all along, right?

Reply:
Sure was, I'm fixing that now as well.